python/tvm/auto_schedule/__init__.py

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
should this folder be named auto_scheduler?

Reply:
The namespace is auto_schedule

Reply:
I think @MarisaKirisame means the namespace should be a noun. So auto_scheduler is better. I think so

Reply:
That also works if we all agree, we can send a followup PR for it.

Reply:
Sent new PR for namespace renaming: https://github.com/apache/incubator-tvm/pull/6059